src/plugins/expressions/public/render.ts

```diff
@@ -152,7 +158,7 @@ export class ExpressionRenderHandler {
 export function render(
   element: HTMLElement,
   data: any,
-  options?: Partial<ExpressionRenderHandlerParams>
+  options?: ExpressionRenderHandlerParams
```

Comment:
why this change?

Reply:
Because `ExpressionRenderHandlerParams` was never used on its own, it was always used as `Partial< ExpressionRenderHandlerParams>`, so I removed `Partial<>` and made the required fields optional instead.